packages/tpetra/core/example/Finite-Element-Assembly/fem_assembly_TotalElementLoop_SP.cpp

```diff
@@ -308,18 +306,37 @@ int main (int argc, char *argv[])
   //Tpetra::MatrixMarket::Writer<matrix_t>::writeSparse(ofs, crs_matrix);
   //ofs.close();
 
+  return 0;
+}
+
+
+int main (int argc, char *argv[]) 
+{
+  int status = EXIT_SUCCESS;
+  
+  // MPI boilerplate
+  Tpetra::initialize(&argc, &argv);
+  COMM_T comm = Tpetra::DefaultPlatform::getDefaultPlatform ().getComm();
```

Comment:
Use a typedef instead of a macro here (see above).  Also, use `Tpetra::getDefaultComm()`; don't use `DefaultPlatform`.  The whole "Platform" idea was Chris Baker's original heterogeneous programming model that wasn't quite so fully baked.